batchservice/src/main/java/nu/yona/server/batch/quartz/SchedulingService.java

```diff
@@ -0,0 +1,56 @@
+package nu.yona.server.batch.quartz;
+
+import static org.quartz.SimpleScheduleBuilder.simpleSchedule;
+import static org.quartz.TriggerBuilder.newTrigger;
+
+import java.util.Date;
+import java.util.Map;
+
+import org.quartz.JobDataMap;
+import org.quartz.JobDetail;
+import org.quartz.JobKey;
+import org.quartz.Scheduler;
+import org.quartz.SchedulerException;
+import org.quartz.Trigger;
+import org.quartz.TriggerKey;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import nu.yona.server.exceptions.YonaException;
+
+@Service
+public class SchedulingService
+{
+	public enum ScheduleGroup
+	{
+		CLEAN_UP, GENERATE, OTHER
```

Comment:
Currently, the only value of this enum used is `OTHER`. Why do you have the other values there? Why do you even use a group name?